File: TF10. METHODS. DEB AND TROUBLESHOOTING CODE - EX (Java)/src/P10_CubeProperties.java
import java.util.Scanner;

public class P10_CubeProperties {
    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        double inputNumber = Double.parseDouble(in.nextLine());
        String cubeProperties = in.nextLine();
        double result;

        switch (cubeProperties)

        {
            case "face":

                result = GetFaceDiagonalsLenght(inputNumber);
                System.out.printf("%.2f", result);
                break;

            case "space":

                result = GetSpaceDiagonalsLenght(inputNumber);
                System.out.printf("%.2f", result);
                break;

            case "volume":

                result = GetVolume(inputNumber);
                System.out.printf("%.2f", result);
                break;

            case "area":

                result = GetSurfaceArea(inputNumber);
                System.out.printf("%.2f", result);
                break;
        }
    }

    private static double GetFaceDiagonalsLenght(double n) {
        return Math.sqrt(2 * Math.pow(n, 2));
    }

    private static double GetSpaceDiagonalsLenght(double n) {
        return Math.sqrt(3 * Math.pow(n, 2));
    }

    private static double GetVolume(double n) {
        return Math.pow(n, 3);
    }

    private static double GetSurfaceArea(double n) {
        return 6 * Math.pow(n, 2);
    }
}

File: TF10. METHODS. DEB AND TROUBLESHOOTING CODE - EX/P10.CubeProperties.cs
using System;

namespace P10.CubeProperties
{
    class Program
    {
        static void Main(string[] args)
        {
            double inputNumber = double.Parse(Console.ReadLine());
            string cubeProperties = Console.ReadLine();

            double result = 0.0;


            switch (cubeProperties)

            {
                case "face":

                    result = GetFaceDiagonalsLenght(inputNumber);
                    Console.WriteLine($"{result:f2}");
                    break;

                case "space":

                    result = GetSpaceDiagonalsLenght(inputNumber);
                    Console.WriteLine($"{result:f2}");
                    break;

                case "volume":

                    result = GetVolume(inputNumber);
                    Console.WriteLine($"{result:f2}");
                    break;

                case "area":

                    result = GetSurfaceArea(inputNumber);
                    Console.WriteLine($"{result:f2}");
                    break;
            }


        }

        static double GetFaceDiagonalsLenght(double n)
        {
            double faceDiagonal = Math.Sqrt(2 * Math.Pow(n, 2));
            return faceDiagonal;
        }

        static double GetSpaceDiagonalsLenght(double n)
        {
            double spaceDiagonal = Math.Sqrt(3 * Math.Pow(n, 2));
            return spaceDiagonal;
        }

        static double GetVolume(double n)
        {
            double volume = Math.Pow(n, 3);
            return volume;
        }

        static double GetSurfaceArea(double n)
        {
            double surfaceArea = 6 * Math.Pow(n, 2);
            return surfaceArea;
        }
    }
}